src/widgets/histogram/content-view.js

```diff
@@ -93,6 +93,10 @@ module.exports = cdb.core.View.extend({
       return;
     }
 
+    if (this._dataviewModel.changed.bins) {
+      this.binsCount = this._dataviewModel.get('bins');
```

Comment:
Originally, when the user zoomed out, I used the `this.binsCount` variable to set the value with the initial value (https://github.com/CartoDB/deep-insights.js/blob/e0695807d3c5b915f8795a4bff41539b1ecad15d/src/widgets/histogram/content-view.js#L436)